tensorflow/python/kernel_tests/scatter_nd_ops_test.py

```diff
@@ -364,6 +364,16 @@ def scatter_nd(self, indices, updates, shape, input_=None):
     del input_  # input_ is not used in scatter_nd
     return array_ops.scatter_nd(indices, updates, shape)
 
+  def testString(self):
+    indices = constant_op.constant([[4], [3], [1], [7]], dtype=dtypes.int32)
+    updates = constant_op.constant(["four", "three", "one", "seven"], dtype=dtypes.string)
+    expected = np.array(["", "one", "", "three", "four", "", "", "seven"])
+    scatter = self.scatter_nd(indices, updates, shape=(8,))
```

Comment:
Need an example testing what happen if a particular value appears twice in indices.

Reply:
@josh11b Hi, thanks for reviewing. I'd like to add more tests,  and could you explain more in details to help me understand it correctly?

Reply:
It would be good to do the second case and just validate that the answer is one of the expected possibilities ("bc" or "cb").

Reply:
@josh11b Thanks, sounds reasonable. I'll let you know when those tests are added.